chia/wallet/wallet_state_manager.py

```diff
@@ -1177,15 +1208,16 @@ async def handle_clawback(
             # Add merkle coin
             await self.coin_store.add_coin_record(coin_record)
             # Add tx record
+            # We use confirmed to indicate if a Clawback transaction is claimable
```

Comment:
Seems like this comment should sit next to the `TransactionRecord` creation and not above the timestamp.
```suggestion
            # We use TransactionRecord.confirmed to indicate if a Clawback transaction is claimable
```